ci/matrix.xml

```diff
@@ -88,18 +88,18 @@
         'set scalaJSStage in Global := FullOptStage' \
         ++$scala testSuite/test &&
     sbtretry 'set Seq(testSuite, noIrCheckTest).map(pr => scalaJSOutputMode in pr := org.scalajs.core.tools.javascript.OutputMode.ECMAScript6)' \
-        'set Seq(testSuite, noIrCheckTest).map(pr => postLinkJSEnv in pr := NodeJSEnv(executable = "/home/jenkins/apps/iojs-2.0/bin/iojs", args = Seq("--harmony-rest-parameters")).value.withSourceMap(false))' \
+        'set Seq(testSuite, noIrCheckTest).map(pr => postLinkJSEnv in pr := NodeJSEnv(executable = "/home/jenkins/apps/node/bin/node", args = Seq("--harmony-rest-parameters")).value.withSourceMap(false))' \
```

Comment:
We should probably first have a PR dedicated to only *removing* `executable =` of all the iojs stuff. Then we can rebase this PR on top of that.